src/activity-functions-function-notation.xml

```diff
@@ -235,17 +237,21 @@
                         <li>
                             <p><m>f(x)</m> <var name="$f" form="popup" /></p>
                         </li>
-                    </ol></p>
+                    </ol>
                 </statement>
             </webwork>
         </exercise>
 
-        <p>The format of function notation works on the basic principle of input and output.</p>
+        <p>Function notation like <m>f(t)</m> or <m>f(x)</m> or <m>f(</m>anything<m>)</m> implies there is some kind of relation, some kind of rule, that takes a piece of information and returns a response or result. The format of function notation works on the basic principle of input and output and we often use the notation when we refer to things we don't yet know.</p>
```

Comment:
`<m>f(</m>anything<m>)</m>` should be `<m>f(\text{anything})</m>`. Otherwise it might line break with "f(" at the end of one line and "anything)" at the start of the next line.